engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreSubmitActor.scala

```diff
@@ -166,7 +166,7 @@ final case class WorkflowStoreSubmitActor(store: WorkflowStore, serviceRegistryA
 
     processSource(_.clearEncryptedValues)(originalSourceFiles) map { sourceFiles =>
       val submissionEvents: List[MetadataEvent] = List(
-        MetadataEvent(MetadataKey(id, None, WorkflowMetadataKeys.SubmissionTime), MetadataValue(OffsetDateTime.now.toString)),
+        MetadataEvent(MetadataKey(id, None, WorkflowMetadataKeys.SubmissionTime), MetadataValue(OffsetDateTime.now)),
```

Comment:
`. toUtcMilliString `?

Reply:
One should not be calling `Object.toString()` on **any** Metadata values as the types are not `value: String` but are `value: Any`.

Meanwhile in this PR, Metadata's `Any` converter has been updated to convert `OffsetDateTime` to the correct format. This should help in the future whenever `OffsetDateTime` instances are passed as metadata values.